shell/common/node_bindings.cc

```diff
@@ -398,13 +398,11 @@ base::FilePath GetResourcesPath() {
 }  // namespace
 
 NodeBindings::NodeBindings(BrowserEnvironment browser_env)
-    : browser_env_(browser_env) {
-  if (browser_env == BrowserEnvironment::kWorker) {
+    : browser_env_{browser_env},
+      uv_loop_{browser_env == BrowserEnvironment::kWorker ? &worker_loop_
```

Comment:
Is there an advantage to inlining here ? I feel the previous version is more readable.

Reply:
The advantage is that it lets us declare `raw_ptr<uv_loop_t> uv_loop_` as `const`, which is nice-to-have for understanding the scope & lifespan of a field.

However I agree the previous version was more concise & easier to read. I'll have another go at it.

Reply:
I used extract-method in 6e372d796f471c28b19801c1c5678c83901345d7 to make the code clearer.